### DISCUSSION TITLE
Lightsys+ compatibility

### DISCUSSION THREAD

**Discussion by floco** [vanackej/risco-mqtt-local#35] [Q&A] [answered]
Hello 
Thanks for this great addon :-)
I see the underlying library is compatible with lightsys but do you know if it's compatible with the plus version (lightsys+) ?
Thanks a lot.

**Comment by vanackej** [comment]
Hi,

Have you tried using the library with lightsys plus ? I don't have one, so I can't test compatibility.
If you try let me know if it works. If it does not work, maybe it can be easily added (providing startup logs)

**Reply by floco** [reply]
Thanks for the super fast reply.
I don't have it right now but planning to buy one. So this was just to check if it would work as your integration is a key reason why I'm considering it 😜 and the return policy of the shop does not seem very flexible. Perhaps others have tested it already ?

**Reply by vanackej** [reply]
The technical baseline for the alarm system seems to be the same as the LightSys 2, with the same panel type (RP432)
It could work out the box with the existing LigthSys super, but no guarantee until someone gives a try.

**Reply by floco** [reply]
That's great 👍 Indeed should be close to the other lightsys. I will let you know the results if I go for it. Thanks so much for this library. Still hesitating between Risco (if I can install it myself in place of my very old Aritech CS-121) and something like Konnected or even ESP DIY.

**Reply by floco** [reply]
Hello @vanackej 

Finally went ahead and purchased the lightsys plus :-)
Here is the startup log
Loading config from: /config/risco-mqtt.json
7/1/2023, 12:10:06 PM [info] autoConnect enabled, starting communication
7/1/2023, 12:10:06 PM [info] Connecting to mqtt server: mqtt://core-mosquitto:1883
7/1/2023, 12:10:06 PM [info] Connected on mqtt server: mqtt://core-mosquitto:1883
7/1/2023, 12:10:06 PM [info] Panel is not connected, waiting
/app/node_modules/@vanackej/risco-lan-bridge/dist/RiscoComm.js:304
                    throw new Error(`Unsupported panel type : ${panelType}`);
                          ^
Error: Unsupported panel type : RP432MP
    at RiscoComm.<anonymous> (/app/node_modules/@vanackej/risco-lan-bridge/dist/RiscoComm.js:304:27)
    at Generator.next (<anonymous>)
    at fulfilled (/app/node_modules/tslib/tslib.js:114:62)

**Reply by floco** [reply, marked as answer]
Sorry, I just realised home assistant now had a native local integration and it supports lightsys+.
Thanks anyway

**Reply by needles77** [reply]
Hello @floco ,

I get a lightsys+ also last month and i add some code on the dependent projet "local-lan-bridge"  from @TJForc for support new lightsys+ on Homebridge plug in. it work nice on home my Homebridge / homekit apple. (i fork the project on my github).
i think i will migrate my risco plugin on home assistant, i try to use the native integration but i can't connect my panel, it work with your risco lightsys +  ?

For the next weeks, i will try the @vanackej  risco-mqtt-local add on. The lightsys + works as same ways than Lightsys 2, just the PanelType string is RP432MP for the LightSys+ , i will add the class for the RP432MP on the risco-lan-bridge from @vanackej  and i will try the mqtt plug in.
I'll keep in touch.
Thank you.

**Reply by floco** [reply]
Hello,

I use the native home assistant integration in local mode.
https://www.home-assistant.io/integrations/risco
Works great :-)

**Reply by needles77** [reply]
Okay ! Nice. I will try ! Thank you

**Reply by markxroberts** [reply]
I'm running a fork of this with some extra functionality.  I've added the LightSys Plus.  You're welcome to try it.

**Reply by needles77** [reply]
Thank you @markxroberts !

I'll integrate your version, huge improve and bug fixing !

**Comment by amitkaf** [comment]
Hello everyone i am a proffesional installer for risco lightsys+ ... i wonder to know if there is a code for node red , because i cant find a pallet on the node red library, i dont use home assistant, it always needed to update ... got tired from it
